pkg/cloud/provider/provider.go

```diff
@@ -242,6 +244,16 @@ func NewProvider(cache clustercache.ClusterCache, apiKey string, config *config.
 			ClusterAccountID: cp.accountID,
 			Config:           NewProviderConfig(config, cp.configFileName),
 		}, nil
+	case opencost.LinodeProvider:
+		log.Info("Found ProviderID starting with \"linode\", using Linode Provider")
+		return &linode.Linode{
+			KubeClient:       kubeClientset,
+			Clientset:        cache,
+			ClusterRegion:    cp.region,
+			ClusterAccountID: cp.accountID,
+			ClusterProjectID: cp.projectID,
```

Comment:
What is Cluster ProjectID here

Reply:
ProjectID is a cluster IDish thing, exactly the cluster name reference from hostname (the closest data to cluster ID represented on the cluster)